Fix:

error: call to undeclared function 'lseek'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared function 'read'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared function 'close'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared function 'write'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
--- texmaker.pro.orig	2021-09-02 13:08:25.000000000 -0500
+++ texmaker.pro	2024-07-12 21:01:31.000000000 -0500
@@ -1129,6 +1129,9 @@
 	trans/texmaker_lv.ts \
 	trans/texmaker_uk.ts
 ################################
+unix {
+DEFINES += HAVE_UNISTD_H
+}
 unix:!macx {
 UI_DIR = .ui
 MOC_DIR = .moc
